README.md

```diff
@@ -1 +1,35 @@
 # ResetPasswordBundle
+
+Worrying about how to deal with users that can't remember their password? We've got you covered! This bundle provides a secure out of the box solution to allow users to reset their forgotten passwords.
+
+## Installation
+
+@TODO - To be added upon release
+
+The bundle can be installed using Composer or the [Symfony binary](https://symfony.com/download):
+
+```
+composer install X/X
```

Comment:
Now we can say `composer require symfonycasts/reset-password-bundle` 

They would technically need `@beta` for now - but that will change soon - so don't mention it.

Reply:
done WOOOHOOO!